src/functions/test_function_documentation_string.py

```diff
@@ -40,3 +40,8 @@ def test_function_documentation_string():
 
     No, really, it doesn't do anything.
     """
+    """
+    this is a documenation string
+    i can write anything here but the idea is to write something that will help
+    another person understand this function.
```

Comment:
understand this function by explaining purpose it serves, what type of arguments it expects and its possible return value.

Reply:
i have updated the docstring and also added another test